.github/fabricbot.json

```diff
@@ -1069,6 +1069,15 @@
               ""
             ]
           },
+          {
+            "label": "Machine Learning PR",
```

Comment:
This isn't the form of label that we typically use in the Azure SDK repositories.    Let's make this "Machine Learning", please.

Reply:
Done. Please review again, @jsquire.